### PR TITLE
Changed 'Edition' to Editing or Edit, etc

### PR DESCRIPTION
**Pull request by cornhundred**
I just changed a few instances of the word 'edtion' to 'editing' or 'edit', which I think is clearer (I did not change the link to the image with Edition name).

### CODE REVIEW

**Comment by fzaninotto** [comment]
I'm not a native English speaker, I don't know if you are either. I'd love a second opinion from a native American English speaker.

**Comment by dervos** [comment]
Editing is correct. Edition is more like a version of something, or like _second edition_.

File: docs/Tutorial.md [hunk 1]
```diff
@@ -222,7 +222,8 @@ When displaying the posts list, the app now fetches related user records, and di
 
 **Tip**: Reference components always pass the data they fetch to a child component, which is responsible for displaying the data.
 
-## Creation and Edition
+## Creation and Editing
```
Comment (fzaninotto) [inline docs/Tutorial.md]: You must change the link in `docs/_layouts/default.html`, too

**Comment by cornhundred** [comment]
I updated the link in `docs/_layouts/default.html` to match the new name in the Tutorial.md and fixed another typo (changed "build" to "built")

**Comment by fzaninotto** [comment]
Thanks!

**Pull request merged**